ldap/servers/plugins/sync/sync_refresh.c

```diff
@@ -101,6 +101,13 @@ sync_srch_refresh_pre_search(Slapi_PBlock *pb)
              */
             session_cookie = sync_cookie_create(pb, client_cookie);
             PR_ASSERT(session_cookie);
```

Comment:
Should probably remove this assert since session_cookie does have a valid reason for being NULL.

Reply:
Correct, I removed it.